### DISCUSSION TITLE
Documentation

### DISCUSSION THREAD

**Discussion by lukeconibear** [duncanwp/ESEm#22] [General]
Hi @duncanwp,

Thanks for creating a fantastic project!

Do you have a preferred way to recreate the docs locally?

Best,
Luke

**Comment by duncanwp** [comment]
Hi Luke, thanks for reaching out, I'm glad you like it!

I use a PyCharm docs configuration but it's all just using Sphinx so I think the following should work:
```
sphinx-build -b html ESEm/docs ESEm/local_docs
```

Note, you'll need the required packages in your environment, but you can install those with:
```
pip install -e .[docs]
```
(assuming you're building a local install for development)
